src/include/feature_vector.h

```diff
@@ -58,14 +59,14 @@ using tdbFeatureVectorRange = tdbColMajorMatrix<T>
       const std::string& attr_name,
       const std::vector<uint64_t>& start,
       const std::vector<uint64_t>& end) {
-    tiledb::Array array(ctx, uri, TILEDB_READ);
+    tiledb::Array array = tiledb_helpers::open_array(ctx, uri, TILEDB_READ);
     tiledb::Query query(ctx, array);
     tiledb::Subarray subarray(ctx, array);
     subarray.set_subarray(start, end);
     query.set_subarray(subarray);
     query.set_layout(TILEDB_ROW_MAJOR);
     query.set_data_buffer(attr_name, buffer);
-    query.submit();
+    tiledb_helpers::submit_query(query);
```

Comment:
We should pass the URI and the function scope (there is a macro `tdb_func__` that can be used to get the function scope.  Right now query doesn't have enough context to be interpretable.

We might even consider passing `tdb_func__` to `open_array`.

Reply:
Done.